app/controllers/course/course_settings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Course::CourseSettingsController < Course::ModuleController
```

Comment:
no. Hound still not following config.

If you want, you can install rubocop locally `gem install rubocop` and run `rubocop` from the project root directory.